### ISSUE TITLE
SwiftMessages child (Date & Time picker) VO accessibility does not work.

### ISSUE THREAD

**Issue by amitcse6** [SwiftKickMobile/SwiftMessages#505]
SwiftMessages child (especially for date and time picker), which I want to display through this repo, is not working properly in voice-over mode. Moreover, I disabled all SwiftMessages views (except the desired child view). Please guide me, anyone, on how can I fix this issue.

**Comment by wtmoose** [comment]
If you still need help with this, reopen and I’ll take a look.
